setup.py

```diff
@@ -4,22 +4,28 @@
 
 setup(
     name='m2m',
-    version='1.0.0',
+    version='2.0.0',
     url='https://github.com/unt-libraries/m2m',
     author='Mark Phillips',
     author_email='mark.phillips@unt.edu',
     license='BSD',
     packages=['m2m'],
     scripts=['m2m/m2m.py'],
     install_requires=['pyuntl'],
+    dependency_links=[
+        'git+https://github.com/unt-libraries/pyuntl.git/@master#egg=pyuntl-2.0.0'
+    ],
```

Comment:
So currently, this can be successfully installed with `python setup.py install`, which reads the `dependency_links`, but not with `pip install .`, which does not. Since we use tox, which installs the package with `pip`, we currently use `requirements.txt` to ensure we grab the correct version of `pyuntl`. That still doesn't allow us to install with `pip install .`, and while that isn't necessary, I propose the following changes, which will allow for both installation methods _and_ allow us to get rid of `requirements.txt` and the related installation in `tox.ini`:

define the URL as a variable above `setup()`:
`pyuntl_url = 'git+https://github.com/unt-libraries/pyuntl.git@master#egg=pyuntl`

then change `install_requires` like so (pip, and by extension, tox, will see the link here and use it):
`install_requires=['pyuntl @' + pyuntl_url]`

and finally, just to remove some duplication:
`dependency_links=[pyuntl_url]`

Now, is this the _best_ method? I kinda doubt it. But I have yet to find something definitive on how best to solve the issue of dependencies outside of PyPI that works with both setup.py and pip. Just for some context, here's a couple SO posts that I keep seeing whenever I try looking into this further:
https://stackoverflow.com/questions/26061610/pip-install-dependency-links
https://stackoverflow.com/questions/32688688/how-to-write-setup-py-to-include-a-git-repo-as-a-dependency

Reply:
Sound like an ok thing to try to me.

Reply:
Works fine. And.. doesn't need pyuntl installation separately in tox either.